app/routes/languages.rb

```diff
@@ -6,9 +6,11 @@ class Languages < Core
       TOPICS = [:about, :exercises, :installing, :tests, :learning, :resources, :help, :launch].freeze
 
       get '/languages' do
-        tracks = X::Track.all
-        active, inactive = tracks.partition(&:active?)
-        planned, inactive = inactive.partition(&:planned?)
+        tracks = Trackler.tracks
+        active, inactive = tracks.partition { |t| t.active? }
+        planned = []
```

Comment:
OK, I've updated trackler and pushed to the base trackler branch. If you want to rebase this onto that, it should let you say something like this:

```
       tracks = Trackler.tracks
        erb :"languages/index", locals: {
          active: tracks.select(&:active?),
          upcoming: tracks.select(&:upcoming?),
          planned: tracks.select(&:planned?),
        }
```